scene/gui/label.cpp

```diff
@@ -748,9 +748,13 @@ void Label::set_text(const String &p_string) {
 	text = p_string;
 	xl_text = atr(p_string);
 	dirty = true;
-	if (visible_ratio < 1) {
-		visible_chars = get_total_character_count() * visible_ratio;
+
```

Comment:
```suggestion
```

Reply:
I follow the style of the functions further up that keep key blocks of code surrounded by 1 line of whitespace. I think this functionality is considerably different from what happens before and after.